src/test/java/org/apache/commons/codec/digest/DigestUtilsTest.java

```diff
@@ -63,23 +67,37 @@ File getTestFile() {
         return testFile;
     }
 
+    RandomAccessFile getTestRandomAccessFile() {
+        return testRandomAccessFileWrapper;
+    }
+
     @Before
     public void setUp() throws Exception {
         new Random().nextBytes(testData);
         testFile = File.createTempFile(DigestUtilsTest.class.getName(), ".dat");
         try (final FileOutputStream fos = new FileOutputStream(testFile)) {
             fos.write(testData);
         }
+
+        testRandomAccessFile = File.createTempFile(DigestUtilsTest.class.getName(), ".dat");
+        try (final FileOutputStream fos = new FileOutputStream(testRandomAccessFile)) {
+            fos.write(testData);
+        }
+        testRandomAccessFileWrapper = new RandomAccessFile(testRandomAccessFile, "rw");
     }
 
     @After
     public void tearDown() {
         if (!testFile.delete()) {
             testFile.deleteOnExit();
         }
+
+        if (!testRandomAccessFile.delete()) {
+            testRandomAccessFile.deleteOnExit();
+        }
     }
 
-    @Test(expected=IllegalArgumentException.class)
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
You have reformatted whitespace for the rest of this class.

It makes it hard to see that you have not actually changed anything below this line other than whitespace. Can you unformat please.

Reply:
It is possible to ignore white-space changes in GitHub reviews: https://github.blog/2018-05-01-ignore-white-space-in-code-review/

Reply:
Fixed.